src/Security/CMSSecurity.php

```diff
@@ -165,14 +165,14 @@ protected function preLogin()
      *
      * @return bool
      */
-    public static function enabled()
+    public function enabled()
     {
         // Disable shortcut
         if (!static::config()->get('reauth_enabled')) {
             return false;
         }
 
-        return count(Security::singleton()->getApplicableAuthenticators(Authenticator::CMS_LOGIN)) > 0;
+        return count($this->getApplicableAuthenticators(Authenticator::CMS_LOGIN)) > 0;
```

Comment:
Might need to talk about this on Monday, but I think the CMS_LOGIN key is probably redundant. We probably over-engineered the authenticator assignment system in the initial implementation.